pkg/utils/configMapUtils.go

```diff
@@ -66,17 +61,16 @@ func (expDetails *ExperimentDetails) ValidateConfigMaps(clients ClientSets) erro
 	return nil
 }
 
-func getExperimentConfigmaps(clients ClientSets, expDetails *ExperimentDetails) ([]v1alpha1.ConfigMap, error) {
+func (expDetails *ExperimentDetails) getExperimentConfigmaps(clients ClientSets) ([]v1alpha1.ConfigMap, error) {
 	chaosExperimentObj, err := clients.LitmusClient.LitmuschaosV1alpha1().ChaosExperiments(expDetails.Namespace).Get(expDetails.Name, metav1.GetOptions{})
 	if err != nil {
 		return nil, errors.Errorf("Unable to get ChaosExperiment Resource, error: %v", err)
 	}
 	experimentConfigMaps := chaosExperimentObj.Spec.Definition.ConfigMaps
-
 	return experimentConfigMaps, nil
 }
 
-func getEngineConfigmaps(clients ClientSets, engineDetails EngineDetails, expDetails *ExperimentDetails) ([]v1alpha1.ConfigMap, error) {
+func (expDetails *ExperimentDetails) getEngineConfigmaps(clients ClientSets, engineDetails EngineDetails) ([]v1alpha1.ConfigMap, error) {
```

Comment:
To make it more explicit, can we name this function as `getConfigMapsFromChaosEngine`